styles.css

```diff
@@ -0,0 +1,598 @@
+  font-size: 36px;
+  padding-top: 20px;
+  color: gray;
+  flex-direction: column;
+}
+
+
+.user-menu {
+  color: white;
+  margin: 0px;
+  width: 200px;
+  flex-direction: column;
+  font-size: 12px;
+  margin: 5px;
+  background-color: inherit;
+  flex-wrap: wrap;
+
+}
+
+.url {
+  padding-left: 3px;
+  padding-top: 3px;
+}
+
+.ninja {
+  background-image: url('./assets/header2_ninja.png');
+}
+
+.ludwig {
+  background-image: url('./assets/header2_ludwig.png');
+}
+
+.ranboolive {
+  background-image: url('./assets/header2_ranboolive.png')
+}
+
+.shroud {
+  background-image: url('./assets/header2_shroud.png');
+}
+
+.summit {
+  background-image: url('./assets/header2_summit.png');
+}
+
+.xqc {
+  background-image: url('./assets/header2_xqc.png');
+}
+
+.trymacs {
+  background-image: url('./assets/header2_trymacs.png');
+}
+
+.nickmercs {
+  background-image: url('./assets/header2_nickmercs.png');
+}
+
+.tfue {
+  background-image: url('./assets/header2_tfue.png');
+  outline: none;
+  border: none;
+}
+
+.hasanabi {
+  background-image: url('./assets/header2_hasanabi.png');
+  outline: none;
+  border: none;
+}
+
+}
+
+
+
+
+
+
+
+
+
+
+
+
+
+
+@media screen and (max-width: 1040px) {
+
+html,
+body
+main {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  width: 100%;
+  font-family: 'Dela Gothic One', cursive;
+  font-family: 'Lato', sans-serif;
+  overflow: scroll;
+
+}
+
+
+article {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  outline: none;
+  border: none;
+
+
+}
+
+button {
+  outline: none;
+  border: none;
+
+}
+
+div {
+
+}
+
+header {
+  background-color: #0AF5E8;
+  height: 15vh;
+  width: 100%;
+  display: flex;
+  outline: none;
+  border: none;
+}
+
+nav {
+  display: flex;
+  height: 15vh;
+  width: 100%;
+
+}
+
+p {
+  display: flex;
+  flex-direction: column;
+  margin: 3px;
+}
+
+section {
+  justify-content: center;
+  display: flex;
+  flex-direction: column;
+  outline: none;
+  border: none;
+
+}
+
+.streamer-name {
+  font-size: 16px;
+  padding-left: 10px;
+  padding-top: 10px;
+
+}
+
+.date-achieved {
+  font-size: 10px;
+  padding-left: 10px;
+  padding-bottom: 15px;
+  margin-top: 0px;
+}
+
+.most-played-game,
+.sub {
+  font-size: 12px;
+  padding-left: 10px;
+  padding-top: 10px;
+  margin-bottom: 0px;
+}
+
+.sub-count {
+  font-size: 24px;
+  padding-left: 10px;
+  padding-bottom: 20px;
+  margin-top: 0px;
+  color: #F50A17;
+
+}
+
+.game-name {
+  padding-left: 10px;
+  font-size: 16px;
+  margin-top: 0px;
+}
+
+
+
+
+.all-time,
+.position {
+  flex-direction: column;
+  margin-bottom: 0px;
+  margin-right: 10px;
+  margin-top: 20px;
+  align-self: flex-end;
+  color: white;
+}
+
+
+
+.position {
+  font-size: 30px;
+  margin-top: 0px;
+  margin-bottom: 30px;
+}
+
+.background {
+  height: 100%;
+  width: 100%;
+  flex-direction: column;
+  background-color: white;
+  color: grey;
+  align-items: center;
+}
+
+
+.card {
+  display: flex;
+  width: 250px;
```

Comment:
This is an example of somewhere that a more responsive metric than px could give you more flexiblity. Just a suggestion because currently the pixels look fine on a few different screen sizes for me.